common/scripts/install-run.js

```diff
@@ -373,7 +373,12 @@ function installAndRun(packageName, packageVersion, packageBinName, packageBinAr
         cwd: process.cwd(),
         env: process.env
     });
-    return result.status;
+    if (result.status !== null) {
```

Comment:
Ideally, we would use `!=` here to ensure that both `null` and `undefined` cases are handled.

Reply:
This script is installed by the `rush` tool, so any feedback would need to be made in the rush repo.